<ui version="4.0" >
 <class>pqRescaleRangeDialog</class>
 <widget class="QDialog" name="pqRescaleRangeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Set Range</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="4" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="RescaleButton" >
       <property name="text" >
        <string>&amp;Rescale</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Maximum</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QLineEdit" name="MaximumScalar" >
     <property name="whatsThis" >
      <string>Enter the new range maximum here.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="MinimumScalar" >
     <property name="whatsThis" >
      <string>Enter the new range minimum here.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Minimum</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>MinimumScalar</tabstop>
  <tabstop>MaximumScalar</tabstop>
  <tabstop>RescaleButton</tabstop>
  <tabstop>CancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
